ldap/servers/slapd/back-ldbm/ldbm_index_config.c

```diff
@@ -78,6 +78,14 @@ ldbm_index_parse_entry(ldbm_instance *inst, Slapi_Entry *e, const char *trace_st
         }
     }
 
+    if (0 == slapi_entry_attr_find(e, "nsSystemIndex", &attr)) {
```

Comment:
I think you may '*is_system_index = PR_FALSE;' by default, just in case the caller does not initialize it.

Reply:
That is already done by the calling function, but I can doubly enforce it in this function.